leader-election-impl/src/main/java/com/palantir/paxos/SqlitePaxosStateLogMigrationState.java

```diff
@@ -52,12 +53,24 @@ private void initialize() {
         execute(Queries::createTable);
     }
 
-    public void finishMigration() {
-        execute(dao -> dao.finishMigration(namespace, useCase));
+    public void migrateToValidationState() {
+        execute(dao -> dao.migrateToVersion(namespace, useCase, States.VALIDATION.schemaVersion));
```

Comment:
Discussed offline: Let's validate that we aren't in migrated state before doing this.